src/main/java/Jeremy.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Scanner;
+
+public class Jeremy {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        TaskList taskList = new TaskList(100);
+        PrintUtils.greeting();
+        PrintUtils.logo();
+
+        String userInput = scanner.nextLine();
+        while (!userInput.equals("bye")) {
+            String[] parts = userInput.split(" ", 2);
+            String commandStr = parts[0];
+            String argument = parts.length > 1 ? parts[1] : "";
+            Command command = Command.fromString(commandStr);
+
+            switch (command) {
```

Comment:
I like that your switch case follows the coding standard and the "case" is flushed with "switch".